LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Sami Ullah
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
```

Comment:
- [ ] According to the project guidelines, we are required to follow the design. Here is the screenshot of the requirements

![Screenshot from 2022-10-12 18-05-42](https://user-images.githubusercontent.com/46388113/195416462-3f463611-cd21-402c-8064-637a923b9a38.png)

Meanwhile our app follows the 1 column layout on mobile devices. Here is a screenshot of the expected layout
![Screenshot from 2022-10-12 18-06-09](https://user-images.githubusercontent.com/46388113/195416767-e913d719-db19-4a43-a88a-30b0c5ed2c41.png)